addons/vehicles/CfgVehicles/Raptor.inc

```diff
@@ -0,0 +1,144 @@
+
+#undef AUTHOR
+#define AUTHOR Demon
+
+class SWS_V_Heli_Raptor_Base : Heli_Attack_02_dynamicLoadout_base_F
+    {
+    ITEM_META(0);
+    displayName = QNAME(Sx-48 Raptor - Base);
+    faction = "SWS_UNSC";
+    side = 1;
+    crew = "B_UNSCAF_Pilot_01";
```

Comment:
Either add "B_UNSCAF" to the requiredAddons array in config.cpp for vehicles or use a pilot AI from the existing deps.

Reply:
Done.